dashboard/client/src/pages/metrics/Metrics.tsx

```diff
@@ -377,6 +421,8 @@ export const Metrics = () => {
   const toParam = to !== null ? `&to=${to}` : "";
   const timeRangeParams = `${fromParam}${toParam}`;
 
+  const refreshParams = refresh !== "" ? `&refresh=${refresh}` : "";
```

Comment:
```suggestion
  const refreshParams = refresh ? `&refresh=${refresh}` : "";
```
Since refresh can also be `null`?

Reply:
That's right, modified.